Tests/NIOTests/ChannelTests.swift

```diff
@@ -83,6 +83,41 @@ class ChannelLifecycleHandler: ChannelInboundHandler {
     }
 }
 
+fileprivate class VerifyConnectionFailureHandler: ChannelInboundHandler {
```

Comment:
would you mind putting this class at the bottom of the file? I don't really like when I need to search for the beginning of the actual test class as that is where Xcode puts the 'run all tests in this class' button. I think our test files should all start with the actual test class and not the helpers.